File: src/main/java/de/silencio/activecraftcore/utils/config/WarpsConfig.java
package de.silencio.activecraftcore.utils.config;

import lombok.Getter;
import org.bukkit.Location;

import java.util.HashMap;

@Getter
public class WarpsConfig extends ActiveCraftConfig {

    private HashMap<String, Location> warps;

    public WarpsConfig() {
        super(new FileConfig("warps.yml"));
    }

    @Override
    protected void load() {
        warps = new HashMap<>();
        fileConfig.getKeys(false).stream()
                .filter(key -> fileConfig.getLocation(key) != null)
                .forEach(key -> warps.put(key, fileConfig.getLocation(key)));
    }
}

File: src/main/java/org/activecraft/activecraftcore/utils/config/WarpsConfig.kt
package org.activecraft.activecraftcore.utils.config

import org.bukkit.Location

class WarpsConfig : ActiveCraftConfig(FileConfig("warps.yml")) {
    var warps: Map<String, Location> by configValue("", mapOf()) // TODO: klappt das wirklich mit ""?
    override fun load() {
        /*warps = HashMap()
        fileConfig.getKeys(false).stream()
            .filter { key: String? -> fileConfig.getLocation(key!!) != null }
            .forEach { key: String -> warps!![key] = fileConfig.getLocation(key)!! }*/
    }
}